index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+</head>
+<body>
+ 
+<header class="navbar">
+    <div class="logo"><a href="#"></a>Eros</div>
+    <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
```

Comment:
- [ ] Hey @Hazard10-eng 👋 Your effort so far is commendable 👍 but the `<a>` tag you added to your logo element is not working. I would suggest you add it like this `<div class="logo"><a href="#">Eros</a></div>` this way it is clickable and your user experience is enhanced.
- [ ] Also, it is vital to have your hamburger menu enclosed in a `<nav>` tag for semantic reasons to improve user experience and accessibility.